### ISSUE TITLE
sphinx doesn't run with jython2.5.1

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#265]
Running Sphinx v0.6.3 (via Ant)

```
 [exec] Exception occurred:

 [exec]   File "C:\jython\jython2.5.1\Lib\site-packages\sphinx-0.6.3-py2.5.egg\sphinx\highlighting.py", line 15, in <module>

 [exec]     import parser

 [exec] ImportError: No module named parser

 [exec] The full traceback has been saved in ....\sphinx-err-o_qfvt.log, if you want to report the issue to the author.
```

I tryed to uncomment the parser import ...

 see http://stackoverflow.com/questions/1085791/does-sphinx-run-on-jython

to run sphinx on jython i had to do some changes:

   1.

```
  in the **highlighting.py** file:

  uncomment import parser [line 15?]

  uncomment the use of parser in PygmentsBridge.try_parse (whole try / except ) [line 159 - 164?]

  add an return True to PygmentsBridge.try_parse

  (try_parse is now useless ?!)
```

   2.

```
  in the **util/__init__.py** adding err.errno == 0 in ensuredir (win32 only)

  if not err.errno == 17 and not err.errno == 0:
```

It worked for me!

Are there any unittests ?

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/265
- Originally reported by: [blauhase](https://bitbucket.org/blauhase)
- Originally created at: 2009-09-28T16:16:06.165

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-11-08 16:01:27+00:00_

I fixed the two points you had in <<cset b0ee52829843b50d40cc4c17290f2670fab6bb2b>>.  The unit tests are in the tests/ subdirectory of the source distribution.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:30+00:00_

Removing version: 0.6.3 (automated comment)
